packages/kit/src/runtime/server/page/render.js

```diff
@@ -159,9 +159,10 @@ export async function render_response({
 
 			${serialized_data
 				.map(({ url, body, json }) => {
-					return body
-						? `<script type="svelte-data" url="${url}" body="${hash(body)}">${json}</script>`
-						: `<script type="svelte-data" url="${url}">${json}</script>`;
+					const attrTag = 'data-type="svelte-data"';
```

Comment:
you could just inline these first two variables. for the third I would try to keep the extra space from making it to the output even if it makes it a bit uglier

```
const attrBody = body ? ` data-body="${hash(body)}"` : '';
return `<script type="application/json" data-type="svelte-data" data-url="${url}"${attrBody}>${json}</script>`;
```

Reply:
I tried to prioritize readability vs terseness. imho this is cleaner especially since Prettier would otherwise add a very awkward linebreak, see https://github.com/sveltejs/kit/blob/a604f8c2369055b75aabef0029b2d0fbeb16385a/packages/kit/src/runtime/server/page/render.js#L163-L165 But if you insist I'll do it the other way.